src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageDetailControlModel.cs

```diff
@@ -214,10 +293,27 @@ public bool IsSelectedVersionInstalled
             {
                 return SelectedVersion != null
                     && InstalledVersion != null
+                    && SelectedVersion?.Range?.OriginalString == InstalledVersionRange?.OriginalString
                     && SelectedVersion.Version == InstalledVersion;
             }
         }
 
+        public bool IsAvailableForInstall
```

Comment:
The name doesn't match the code in my mind. If the selected version is installed already, then it shouldn't also be available for installing (again).